modules/ca/main.tf

```diff
@@ -342,20 +343,37 @@ resource "aws_ecs_task_definition" "ca_task_definition" {
        "privileged": false,
        "name": "nerves_hub_ca",
        "environment": [
-         {
-           "name": "ENVIRONMENT",
-           "value": "${terraform.workspace}"
-         }
+         ${local.ecs_shared_env_vars}
        ],
+       "secrets": [
+          ${local.ecs_shared_ssm_secrets},
+          ${local.ecs_dac_shared_ssm_secrets}
```

Comment:
Some of the acronyms I'm not sure I am connecting the dots on 😅. What's dac in this case?

Reply:
Device, API, Ca

Reply:
Sorry - will call that out if we end up leaving this approach